compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STReAtomCharOrEscapeNode.java

```diff
@@ -43,7 +43,7 @@ public class STReAtomCharOrEscapeNode extends STNode {
     STReAtomCharOrEscapeNode(
             STNode reAtomCharOrEscape,
             Collection<STNodeDiagnostic> diagnostics) {
-        super(SyntaxKind.RE_CHAR_ESCAPE, diagnostics);
+        super(SyntaxKind.RE_LITERAL_CHAR_DOT_OR_ESCAPE, diagnostics);
```

Comment:
Seems this change is not done via the treegen.  Change is not reflected in the `syntax_tree_descriptor.json` file.